CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @digitalbazaar/http-signature-zcap-verify ChangeLog
 
+## 11.1.1 - 2024-04-xx
+
+### Fixed
+- Fixed an issue where verification would fail if the url used to create the
+  `invocationTarget` when checking the proof contains a colon.
```

Comment:
```suggestion
- Fixed an issue where verification would fail if a relative url used to
  create the `invocationTarget` when checking the proof contains a
  colon. This allows URLs that contain colon that should be
  URI-encoded but are not, which is bad practice / non-conformant,
  but often allowed in applications.
```